release-please-config-submodules.json

```diff
@@ -0,0 +1,335 @@
+{
+  "release-type": "go-yoshi",
+  "separate-pull-requests": true,
+  "include-component-in-tag": true,
+  "tag-separator": "/",
+  "packages": {
```

Comment:
Ditto for management of this set.

Reply:
Yes, this will need to be updated for new modules.